docs-2.0/3.ngql-guide/12.vertex-statements/4.delete-vertex.md

```diff
@@ -30,9 +42,9 @@ This query shows that you can use `DELETE VERTEX` together with pipe to delete v
 nebula> GO FROM "player100" OVER serve WHERE properties(edge).start_year == "2021" YIELD dst(edge) AS id | DELETE VERTEX $-.id;
 ```
 
-## Delete the process and the related edges
+## Delete the process
 
-Nebula Graph traverses the incoming and outgoing edges related to the vertices and deletes them all. Then Nebula Graph deletes the vertices.
+Nebula Graph deletes the vertices, all edges (incoming and outgoing edges) of the target vertex will become dangling edges.
```

Comment:
1. The title and the sentence have grammar issues.
2. We wrote this section because there was a process (basically 2 steps): delete the vertex, then delete the edges. But now the system only deletes the vertex. Do we still need this section? Or maybe we can write clearly when `WITH EDGE` is added, there is still a process.